File: app/src/main/java/com/example/refshoppe/MainActivity.java
package com.example.refshoppe;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import com.google.android.material.bottomnavigation.BottomNavigationView;

import androidx.navigation.NavController;
import androidx.navigation.Navigation;
import androidx.navigation.ui.NavigationUI;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        BottomNavigationView navView = findViewById(R.id.navigation);
        NavController navController = Navigation.findNavController(this, R.id.fragment_container);
        NavigationUI.setupWithNavController(navView, navController);
    }
}

File: kotlin/app/src/main/java/com/fullstorydev/shoppedemo/MainActivity.kt
package com.fullstorydev.shoppedemo

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import com.fullstory.FS
import com.fullstory.FSOnReadyListener
import com.fullstory.FSSessionData
import com.fullstorydev.shoppedemo.ui.main.MainFragment

class MainActivity : AppCompatActivity(), FSOnReadyListener {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.main_activity)
        if (savedInstanceState == null) {
            supportFragmentManager.beginTransaction()
                    .replace(R.id.container, MainFragment.newInstance())
                    .commitNow()
        }

        FS.setReadyListener(this)
    }

    override fun onReady(sessionData: FSSessionData?) {
        Log.d("MainActivity", "SessionURL is: " + FS.getCurrentSessionURL())
    }
}
